pkg/model/parser.go

```diff
@@ -239,7 +239,9 @@ func (p *NSXConfigParser) getRuleConnections(rule *collector.Rule) *netset.Trans
 		// array. Error will be thrown if ANY is used in conjunction with other values.
 		Services []string `json:"services,omitempty" yaml:"services,omitempty" mapstructure:"services,omitempty"`
 	*/
-	if slices.Contains(rule.Services, anyStr) {
+	// in case both services and serviceEntries are empty, all connection is allowed
+	// otherwise, we union the connections of all the services and the service entries
+	if (len(rule.Services) == 0 || slices.Contains(rule.Services, anyStr)) && len(rule.ServiceEntries) == 0 {
```

Comment:
it's not a union, if services contains "any" and service_entires is not empty, then only service_entires should be treated? (the "any" should be ignored?)

Reply:
you are right, and the implementation is according.
I will elaborate the comment

Reply:
fixed